### PR TITLE
Add David Hadas to TOC charter

### PR DESCRIPTION
**Pull request by dsimansk**
# Changes

- Add David Hadas to TOC charter to fill vacant seat after Zbynek Roubalik 

@davidhadas on behalf of Knative community, let me welcome you to the Technical Oversight Committee. I hope you will like it and bring new & fresh ideas. :)

/cc @knative/technical-oversight-committee
/cc @knative/steering-committee

### CODE REVIEW

File: TECH-OVERSIGHT-COMMITTEE.md [hunk 1]
```diff
@@ -94,10 +94,11 @@ The current members of the TOC are shown below.
 
 | &nbsp;                                                         | Member           | Company    | Profile                                              | Term Start | Term End |
 | -------------------------------------------------------------- | ---------------- | ---------- | ---------------------------------------------------- | ---------- | --------
+| <img width="30px" src="https://github.com/davidhadas.png">     | David Hadas      | IBM        | [@davidhadas](https://github.com/davidhadas)         | 2023-09-12 | 2024     |
```
Comment (dsimansk) [inline TECH-OVERSIGHT-COMMITTEE.md:97]: @davidhadas pls check the details, email etc. Thanks!

**Comment by dsimansk** [comment]
To confirm details like email, affiliation, etc. 
/hold

File: TECH-OVERSIGHT-COMMITTEE.md (hunk 1 above)
Comment (davidhadas) [reply TECH-OVERSIGHT-COMMITTEE.md:97]: all is fine - thanks.

**Comment by davidhadas** [comment]
Thank you. 
Happy to join.

**Comment by csantanapr** [comment]
Thank you @dsimansk and welcome @davidhadas  🎉

**Pull request merged**